friendshipper/src-tauri/src/repo/operations/status.rs

```diff
@@ -156,6 +156,14 @@ where
             status.lock_user.clone_from(&self.github_username);
             status.locks_ours = locks.ours;
             status.locks_theirs = locks.theirs;
+
+            event!(tracing::Level::INFO, "locks_user: {:?}", status.lock_user);
+            event!(tracing::Level::INFO, "locks_ours: {:?}", status.locks_ours);
+            event!(
+                tracing::Level::INFO,
+                "locks_theirs: {:?}",
+                status.locks_theirs
+            );
```

Comment:
if these work for you they work for me, but also you can do fields if you want to be able to query em: https://docs.rs/tracing/latest/tracing/macro.event.html#examples

Reply:
this also might allow you to consolidate into a single event